<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutBorneStationnementUI</class>
 <widget class="QDialog" name="AjoutBorneStationnementUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutBorneStationnementUI</string>
  </property>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant de la voie publique:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idVoiePublique">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateNaissance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom topographique :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AJout d’une borne de stationnement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_codeDepartement">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Côté de la rue :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_quantite">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de la borne :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_salaire">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_commission">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="identifiantBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomTopographique">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_longitude">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>14</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_latitude">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>14</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_coteRue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="numBorne">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AjoutBorneStationnementUI</receiver>
   <slot>validerBorneStationnement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerBorneStationnement()</slot>
 </slots>
</ui>
